README.md

```diff
@@ -112,7 +112,8 @@ especially in the context of studying a [BSc Computer Science offered by Goldsmi
 
 <details>
 <summary><strong>Level 4 (year 1+)</strong></summary>
-
+  
+<div class = "table-container">
```

Comment:
I know it makes no difference, but I'm a fan of consistency when it comes to formatting. :smiley: 

```suggestion
<div class="table-container">
```

Reply:
Haha, I'll make sure to look at previous examples and keep formatting consistent to make it easier next time 😄